src/preferences/dialog/dlgpreflibrary.cpp

```diff
@@ -495,6 +513,15 @@ void DlgPrefLibrary::slotSearchDebouncingTimeoutMillisChanged(int searchDebounci
     WSearchLineEdit::setDebouncingTimeoutMillis(searchDebouncingTimeoutMillis);
 }
 
+void DlgPrefLibrary::searchHistoryOptionsChanged() {
```

Comment:
Since this is not a slot, name the function according to what it does, not when it is called. 
```suggestion
void DlgPrefLibrary::updateSearchLineEditHistoryOptions() {
```